h/accounts/views.py

```diff
@@ -219,7 +219,7 @@ def post(self):
         self._send_forgot_password_email(user)
 
         return httpexceptions.HTTPFound(
-            self.request.route_path('reset_password'))
+            self.request.route_path('account/reset'))
```

Comment:
`request.route_path` [takes the route name as the first argument](http://docs.pylonsproject.org/projects/pyramid/en/latest/api/request.html#pyramid.request.Request.route_path), so this should be `account_reset`.